docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -153,9 +191,9 @@ export default function GlobalCssSlider() {
 }
 ```
 
-### Controlling priority ⚠️
+### CSS injection order ⚠️
 
-**Note:** Most CSS-in-JS solutions inject their styles at the bottom of the HTML `<head>`. If you don't want to mark style attributes with **!important**, you need to change the CSS injection order. Here's a demo of how it can be done for the default style engine - emotion:
+**Note:** Most CSS-in-JS solutions inject their styles at the bottom of the HTML `<head>`, which make Material-UI take precedence over your `<style>`. To remove the need to mark your style attributes with **!important**, you need to change the CSS injection order. Here's a demo of how it can be done for the default style engine - emotion:
```

Comment:
Should this be made consistent with the preceding version?

Reply:
Yep, updated all "CSS injection order" sections to match the new one